components/calendar/src/cal/chinese.rs

```diff
@@ -158,6 +169,12 @@ pub trait Rules: Clone + core::fmt::Debug + crate::cal::scaffold::UnstableSealed
 /// required before 1900, clients can implement their own [`Rules`] type
 /// using data such as from the excellent compilation by [Yuk Tung Liu].
 ///
+/// The precise behavior of this calendar may change in the future if:
+/// - New ground truth in China is established, either through published government sources or most almanacs
+/// - Future data is published that we wish to incorporate
```

Comment:
this is the same as either "new ground truth" or "we decide to expand the range"